lib/modules/us_core_guidance/clinicalnotes_sequence.rb

```diff
@@ -19,152 +29,143 @@ class USCoreR4ClinicalNotesSequence < SequenceBase
-              @document_attachments.attachment[content.attachment.url] = document.id
-            end
+        diagnostic_reports&.each do |report|
+          report&.presentedForm&.select { |attachment| !self.report_attachments.attachment.key?(attachment&.url) }&.each do |attachment|
+            self.report_attachments.attachment[attachment.url] = report.id
           end
         end
       end
 
-      test 'Server shall have Consultation Notes' do
+      test :have_consultation_note do
         metadata do
-          id '02'
+          id '01'
+          name 'Server shall have Consultation Note from DocumentReference search by patient+type'
           link 'https://www.hl7.org/fhir/us/core/clinical-notes-guidance.html'
           description %(
           )
           versions :r4
         end
 
-        test_clinical_notes_document_reference('11488-4')
+        test_clinical_notes_document_reference('http://loinc.org|11488-4')
       end
 
-      test 'Server shall have Discharge Summary' do
+      test :have_discharge_summary do
         metadata do
-          id '03'
+          id '02'
+          name 'Server shall have Discharge Summary from DocumentReference search by patient+type'
           link 'https://www.hl7.org/fhir/us/core/clinical-notes-guidance.html'
           description %(
           )
           versions :r4
         end
 
-        test_clinical_notes_document_reference('18842-5')
+        test_clinical_notes_document_reference('http://loinc.org|18842-5')
       end
 
-      test 'Server shall have History and Physical Note' do
+      test :have_history_note do
         metadata do
-          id '04'
+          id '03'
+          name 'Server shall have History and Physical Note from DocumentReference search by patient+type'
           link 'https://www.hl7.org/fhir/us/core/clinical-notes-guidance.html'
           description %(
           )
           versions :r4
         end
 
-        test_clinical_notes_document_reference('34117-2')
+        test_clinical_notes_document_reference('http://loinc.org|34117-2')
       end
 
-      test 'Server shall have Procedures Note' do
+      test :have_procedures_note do
         metadata do
-          id '05'
+          id '04'
+          name 'Server returns Procedures Note from DocumentReference search by patient+type'
           link 'https://www.hl7.org/fhir/us/core/clinical-notes-guidance.html'
           description %(
           )
           versions :r4
         end
 
-        test_clinical_notes_document_reference('28570-0')
+        test_clinical_notes_document_reference('http://loinc.org|28570-0')
       end
 
-      test 'Server shall have Progress Note' do
+      test :have_progress_note do
         metadata do
-          id '06'
+          id '05'
+          name 'Server shall have Progress Note from DocumentReference search by patient+type'
```

Comment:
Please make language consistent between all test names if they are doing consistent things.  I'm noticing shall language mixed with "server returns...".  Let's take out the shall language.